src/main/java/com/exasol/projectkeeper/ProjectFilesFitter.java

```diff
@@ -0,0 +1,75 @@
+package com.exasol.projectkeeper;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.StandardCopyOption;
+import java.util.List;
+
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugin.logging.Log;
+
+/**
+ * This class fits the projects file structure.
+ */
+public class ProjectFilesFitter extends AbstractProjectFilesValidator {
```

Comment:
Why does the Filler extend validator? Sounds a little bit wrong

Reply:
renamed it